config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+EG7slYG86cPFf9TUEDK2oSMn1djrkhHAEC0H8XmriCbnBIGBBEDdUeRkezzSutBmfjc0PdRdCwq7waqpPvucA/XjaWlClfxvnBpeS4aYRE7WeGJylUDkpmoJ1Nl/UEXmFAsVpaAqFEPT58DwuYoYWtgh3KVKkyQMyfDOP8ZeUOPTzjiGCcpteuoQwP4uX/U8TcivO6dt7OQYAcweh9z0VpWTYynGqOSpZUlqoU43CdnerN7KULy8RpEMPbjKr0E+95qQbMObK4n/k6+JfYzH+qhH8k5ZJwnSKlah1ccJb5Yd4nxqhgveTFSNHvu3fmXOr3nBHIuldwbhcg1VA2N4uHuzVsQ7sD3TGO1Z8c/i+SmZ72yjeKs9vm5c1N2UhsnVGzLccCi3+JwPp7+hupF1soAvsES0aTHrqPWt--+wLtIgFWsx7S75Ti--0+SJkw2DgcBL0iXIQ+/Idg==
```

Comment:
You should create `credentials.yml.enc .example` file and push to git.
Add this file to gitignore and run `git rm --cached config/credentials.yml.enc` to untrack it